README.md

```diff
@@ -81,8 +87,13 @@ The following are examples of valid sentences:
 Write the BNF (not EBNF) description for this language.
 
 ## A2.1
+<sentence>    ::= "The"|"the" <adjectives> <nouns> <verbs> <adverbs> "."
+
+<adjectives>  ::= "lazy"|"smelly"|""
+<nouns>       ::= "dog"|"cat"
+<verbs>       ::= "ate"|"ran"
+<adverbs>     ::= "slowly"|"noisily"|""
```

Comment:
# 4
A technicality, but line 90 says that a sentence can either be the word "The" or it can be _"the" <adjectives> <nouns> <verbs> <adverbs> "."_  The | operator has the lowest precedence. No deduction for that.

However, your grammar only allows for zero or one adjectives.